modules/gitops-verifying-argo-cd-self-healing-behavior.adoc

```diff
@@ -0,0 +1,32 @@
+[id="verifying-argo-cd-self-healing-behavior_{context}"]
+= Verifying Argo CD self-healing behavior
```

Comment:
I suggest making this module part of this assembly: https://deploy-preview-30615--osdocs.netlify.app/openshift-enterprise/latest/cicd/gitops/configuring-argo-cd-to-recursively-sync-a-git-repository-with-your-application.html

Reply:
I think it makes sense to add it to both asseblies.